activities-async/workflow.go

```diff
@@ -0,0 +1,30 @@
+package activities_async
+
+import (
+	"go.temporal.io/sdk/workflow"
+	"time"
+)
+
+func AsyncActivitiesWorkflow(ctx workflow.Context) (res string, err error) {
+	selector := workflow.NewSelector(ctx)
+	var res1 string
+	var res2 string
+	ao := workflow.ActivityOptions{
+		StartToCloseTimeout: 10 * time.Second,
+	}
+	ctx = workflow.WithActivityOptions(ctx, ao)
+	fut1 := workflow.ExecuteActivity(ctx, SayHello, "Temporal")
+	fut2 := workflow.ExecuteActivity(ctx, SayGoodbye, "Temporal")
+	selector.AddFuture(fut1, func(future workflow.Future) {
+		err = future.Get(ctx, &res1)
+	})
+	selector.AddFuture(fut2, func(future workflow.Future) {
+		err = future.Get(ctx, &res2)
+	})
+	selector.Select(ctx)
+	selector.Select(ctx)
```

Comment:
If an `err` was reported by the first select, you have swallowed it

Reply:
Fixed with https://github.com/temporalio/samples-go/pull/286/commits/7156e315a7a30177366bdb2c5b692f783b789894